tests/model_hub_tests/tensorflow/test_tf_convert_model.py

```diff
@@ -24,6 +26,51 @@ def is_hf_link(link: str):
     return link.startswith("hf_")
 
 
+def get_output_signature_from_keras_layer(model):
+    try:
+        from openvino.frontend.tensorflow.utils import trace_tf_model_if_needed
+        traced_model = trace_tf_model_if_needed(model, None, None, None)
+        return traced_model.structured_outputs
+    except:
+        return None
+
+
+def tf_tensor_to_numpy(tensor):
+    if isinstance(tensor, tf.Tensor):
+        return tensor.numpy()
+    elif isinstance(tensor, (list, tuple)):
+        res = []
+        for elem in tensor:
+            res.append(tf_tensor_to_numpy(elem))
+        if isinstance(tensor, list):
+            return res
+        else:
+            return tuple(res)
+    elif isinstance(tensor, dict):
+        res = {}
+        for out_name, out_value in tensor.items():
+            res[out_name] = tf_tensor_to_numpy(out_value)
+        return res
+    raise Exception("Unknown output type of original FW inference result: {}".format(type(tensor)))
```

Comment:
please put this into `model_hub_tests.tensorflow.utils.py` and rename it to `unpack_tf_result`

Reply:
Done.